### PR TITLE
Log fewer lines when making requests to titus-isolate

### PR DESCRIPTION
**Pull request by rgulewich**
The `Titus Isolate did not return code 200` message gets logged a *lot*, and can be potentially misleading.  Log it less.

### CODE REVIEW

File: executor/runtime/docker/docker_isolate.go [hunk 1]
```diff
@@ -82,6 +87,11 @@ func workloadIsolated(ctx context.Context, taskID, host string) bool {
...
 	}
 
+	if resp.StatusCode == 404 {
+		// Only log unexpected status codes below
+		return false
+	}
+
 	logrus.WithFields(map[string]interface{}{
 		"statusCode": resp.StatusCode,
 		"status":     resp.Status,
```
Comment (solarkennedy) [inline executor/runtime/docker/docker_isolate.go:97]: Can you make it say `Titus isloate didn't return 200 after N tries?` ?

File: executor/runtime/docker/docker_isolate.go [hunk 2]
```diff
@@ -82,6 +87,11 @@ func workloadIsolated(ctx context.Context, taskID, host string) bool {
 		return true
 	}
 
+	if resp.StatusCode == 404 {
+		// Only log unexpected status codes below
```
Comment (solarkennedy) [inline executor/runtime/docker/docker_isolate.go:91]: Is this expected? why would we get a 404 after all these attempts?

**Review by solarkennedy — APPROVED** [review]

**Review by gabrielhartmann — APPROVED** [review]

**Comment by sargun** [comment]
Why is it happening so much? Why is it misleading?

**Comment by gabrielhartmann** [comment]
titus-executor polls titus-isolate very quickly. The current logging looks
like there's something wrong when it's really by design behavior.

File: executor/runtime/docker/docker_isolate.go (hunk 2 above)
Comment (rgulewich) [reply executor/runtime/docker/docker_isolate.go:91]: 404 is isolate's response when it hasn't completed the isolation yet.

File: executor/runtime/docker/docker_isolate.go (hunk 1 above)
Comment (rgulewich) [reply executor/runtime/docker/docker_isolate.go:97]: Yeah, I added the line about tries above.  I don't think it's important to log what try an unexpected request occurred on - only what was returned.

**Comment by sargun** [comment]
Can we add a comment indicating that this poll loop is non-blocking on the titus isolate side?

**Comment by rgulewich** [comment]
@sargun - https://github.com/Netflix/titus-executor/pull/521

**Pull request merged**